components/org.wso2.carbon.status.dashboard.web/src/overview/WorkerOverview.jsx

```diff
@@ -88,25 +102,29 @@ export default class WorkerOverview extends React.Component {
             .then((response) => {
                 this.setState({
                     clustersList: response.data,
-                    isApiCalled: true
+                    isApiCalled: true,
+                    statusMessage:!WorkerOverview.hasWorkers(this.state.clustersList) ? "Currently there are no" +
+                        " workers to display" : ''
                 });
             }).catch((error) => {
-            let re = /The session with id '((?:\\.|[^'])*)'|"((?:\\.|[^"])*)" is not valid./;
-            let found = error.response.data.match(re);
-            if (found != null) {
-                this.setState({
-                    isApiCalled: true,
-                    sessionInvalid: true,
-                    statusMessage: "User session Invalid, Please Login"
-                })
-            } else {
+            let message;
+            if(error.response != null){
```

Comment:
space before {

Reply:
check other places also

Reply:
Fixed in b4a2bcf444d25031775a679bad76798d70ce7d5b